treeherder/seta/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import url
```

Comment:
I think it makes more sense to just mix this code into the same webapp code we have under `treeherder/webapp/api`. Also, you should use django rest framework here, which will create a self documenting API (and is also just plain convenient). See `treeherder/webapp/api/performance_data.py` for an example of how I did so with Perfherder.